frontend/pages/sacfairguide.tsx

```diff
@@ -0,0 +1,285 @@
+import s from 'styled-components'
```

Comment:
We may want to name this something else, since we'll also have a guide for students. Maybe like `/sacfair/student` and `/sacfair/officer` or a guides folder or something? This works, just trying to future proof it when we'll have to add another guide.